tamil/isis/index.php

```diff
@@ -10,7 +10,7 @@
 ?>
 <h2 class="red underline large">Keyman for Tamil (ISIS)</h2>
 <p>
-    Type in Tamil on iPhone, Windows and Android. Our Tamil keyboards works with Microsoft Word, Photoshop, Facebook, Twitter, email and thousands of other applications. Free trial available!
+    Type in Tamil on iPhone, Windows and Android. Our Tamil keyboards works with Microsoft Word, Photoshop, Facebook, Twitter, email and thousands of other applications.
 </p>
```

Comment:
You seem to have missed the download box on this page?

Reply:
No, the keyboard sources don't claim that they support Linux

Reply:
@mcdurdin `isis_tamil` is under `legacy` - does this still work on Linux?

Reply:
Ugh yeah, the isis keyboards are pretty much deprecated. Don't worry about it too much.